docs/src/main/asciidoc/security-built-in-authentication.adoc

```diff
@@ -23,7 +23,29 @@ If the user name is `Alice` and the password is `secret`, the HTTP authorization
 ====
  
 The Basic Authentication mechanism does not provide confidentiality protection for the transmitted credentials. The credentials are merely encoded with Base64 when in transit and not encrypted or hashed in any way. Therefore, Basic Authentication is used with HTTPS to provide confidentiality.
- 
+
+There are a few limitations associated with Basic Authentication, which include:
```

Comment:
```suggestion
Basic Authentication is a well-specified challenge/response scheme that all browsers and most Web servers understand. The main benefit of this method is its ability to work through proxy servers. However, there are a few limitations associated with Basic Authentication, which include:
```

Reply:
Thanks for sharing this. 
Added this information.